python/tvm/auto_scheduler/measure.py

```diff
@@ -664,15 +665,13 @@ def local_build_worker(args):
     )
     build_func = BuildFunc.build_func
 
-    res = call_func_with_timeout(timeout, _timed_func, args=(inp, build_func, verbose))
```

Comment:
Because you are no longer using `timeout` in this function, can you remove it from the arguments (and where it is passed in below -- line 710).

Reply:
`timeout` is still used in line 674 for building the result if an exception is thrown. Would that be okay if we keep this argument here?

Reply:
Oh that makes sense. I missed that usage.